test/testdrive/proto-billing.td

```diff
@@ -27,24 +27,10 @@ interval_end    YES text
 interval_start  YES text
 records         YES jsonb
 
-# TODO: these should be fully json
-> SELECT * FROM billing WHERE id = 1
+> SELECT * FROM billing
+0 0                   0                   []
 1 2020-01-01_00:00:00 2020-01-01_00:00:09 []
-
-> SELECT id, interval_end, interval_start FROM billing WHERE id = 2
-2 2020-01-01_00:00:19 2020-01-01_00:00:10
-
-> SELECT records->0->'value', records->1->'value' FROM billing WHERE id = 2
-25.0 125.0
-
-> SELECT
-    records->0->'measurements'->0->'measured_value',
-    records->1->'measurements'->1->'measured_value'
-  FROM billing WHERE id = 2
-5.0 256.0
-
-> SELECT records->0->>'meter', records->1->>'meter' FROM billing WHERE id = 2
-user user
```

Comment:
I kind of liked having these here: we might have enough other tests that it's reasonable to get rid of these, but I don't think that we were testing this level of nesting of things, so I kind of liked it as an example/proof of work.

If you feel like there's enough other testing/examples of this then this is fine as is.

Reply:
👍 sure, happy to leave this in. Restored and merging.